var/spack/repos/builtin/packages/py-future/package.py

```diff
@@ -21,55 +23,14 @@ class PyFuture(PythonPackage):
     depends_on("py-setuptools", type="build")
 
     @property
-    def import_modules(self):
-        modules = [
-            "copyreg",
-            "_thread",
-            "past",
-            "past.types",
-            "past.translation",
-            "past.utils",
-            "past.builtins",
-            "reprlib",
-            "html",
-            "builtins",
-            "http",
-            "_dummy_thread",
-            "queue",
-            "xmlrpc",
-            "libfuturize",
-            "libfuturize.fixes",
-            "future",
-            "future.moves",
-            "future.moves.test",
-            "future.moves.urllib",
-            "future.moves.html",
-            "future.moves.http",
-            "future.moves.dbm",
-            "future.moves.xmlrpc",
-            "future.types",
-            "future.tests",
-            "future.utils",
-            "future.builtins",
-            "future.backports",
-            "future.backports.test",
-            "future.backports.urllib",
-            "future.backports.html",
-            "future.backports.http",
-            "future.backports.xmlrpc",
-            "future.backports.email",
-            "future.backports.email.mime",
-            "future.standard_library",
-            "libpasteurize",
-            "libpasteurize.fixes",
-            "socketserver",
-            "_markupbase",
-        ]
+    def skip_modules(self):
+        modules = []
 
-        if "platform=windows" in self.spec:
-            modules.append("winreg")
```

Comment:
Shouldn't we still be excluding this on other platforms?

Reply:
I thought so as well, but on my fedora all versions build just fine with `--test=root` even without it. So it seems that it is not needed.